File: src/test/java/no/nav/fo/veilarbregistrering/bruker/pdl/PdlOppslagConfig.java
package no.nav.fo.veilarbregistrering.bruker.pdl;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class PdlOppslagConfig {

    private final String PDL_PROPERTY_NAME = "PDL_URL";

    @Bean
    PdlOppslagClient pdlOppslagService() {
        return new PdlOppslagClient(null, null) {
        };
    }
}

File: src/test/java/no/nav/fo/veilarbregistrering/bruker/pdl/PdlOppslagConfig.kt
package no.nav.fo.veilarbregistrering.bruker.pdl

import no.nav.fo.veilarbregistrering.bruker.PdlOppslagGateway
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class PdlOppslagConfig {
    @Bean
    fun pdlOppslagClient() = PdlOppslagClient(null, null)

    @Bean
    fun pdlOppslagGateway(pdlOppslagClient: PdlOppslagClient): PdlOppslagGateway = PdlOppslagGatewayImpl(pdlOppslagClient)
}